package.json

```diff
@@ -32,8 +32,8 @@
   },
   "devDependencies": {
     "typescript": "^2.2.1",
-    "node": "^8.0.19",
```

Comment:
same here.  What purpose is this line serving?

Reply:
See comment [here](https://github.com/Microsoft/typed-rest-client/pull/24).